src/shadowbox/docker/Dockerfile

```diff
@@ -13,10 +13,10 @@
 # limitations under the License.
 
 # See versions at https://hub.docker.com/_/node/
-FROM node:8.11.3-alpine
+FROM node:8.14.0-alpine
```

Comment:
Let's update the label below, too, or kill it - the base image includes an `NODE_VERSION` environment variable.

Reply:
Good catch. I've updated it.